pkg/dispatcher/operator/workers/controller_test.go

```diff
@@ -10,6 +10,36 @@ import (
 	"github.com/go-redis/redis"
 )
 
+func TestDeleteCalculation(t *testing.T) {
```

Comment:
This test doesn't test anything.

Reply:
I know, just wanted to know what you think about the delete function. Still not sure about it.

Reply:
Well, the test that you have to write is more complicated than the actual code. You will have to make sure that the feature you are adding works. When a [pod doesn't exist](https://github.com/vega-project/ccb-operator/blob/master/pkg/dispatcher/operator/workers/controller.go#L141-L143) we want two things
- Find the calculation that was assigned to that pod and it's in the processing phase.
- Knowing the `teff` and `logG` values of the calculation we need to change the database as well by deleting the status field of that particular object in Redis DB.

Your function will use all those 3 clients (pod, calculation, and Redis clients) and the test needs to use fake clients in order to test your data.

[This is how you mock a Redis DB server](https://github.com/vega-project/ccb-operator/blob/master/pkg/dispatcher/operator/workers/controller_test.go#L14)

For fake pod client check [this](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/client/fake)

For calculations fake client [this](https://github.com/vega-project/ccb-operator/blob/master/pkg/client/clientset/versioned/fake/clientset_generated.go#L36)